src/components/Input/Input.ts

```diff
@@ -21,6 +26,21 @@ export class CandyInput extends LitElement {
   @property({ type: String })
   shortCut = "⌘K";
 
+  @property({ type: String })
+  get value() {
+    return this._value;
+  }
+  set value(value) {
+    this._value = value;
+    if (this._input) {
```

Comment:
I'm not sure if it's really relevant but you may call a function here called _checkInput_ that'll just check if input is different from _undefined_ now. But maybe later, we'll need to check over properties to avoid injections (it may be overkilled, I confess)

Reply:
At line 17, input is declared using annotation @query("input") which search for the input. It is declared as _input! to forbid variable to be undefined (I can't find this answer on the web but it seems to compile and I am following your words (kotlin)

Reply:
Yes but I was about checking other things that if the input is undefined. I was thinking about checking if the inputs doesn't contain injections.

Reply:
Oh okay, I think it should be seen on another PR related to security